docs/topics/components-stability.md

```diff
@@ -71,9 +71,10 @@ We make sure to document precisely which subcomponents are not stable. We also d
 | Kotlin/JS (IR-based)                                  | Stable       | 1.8                      |                                                                            |
 | Kotlin/Native Runtime                                 | Stable       | 1.9.20                   |                                                                            |
 | Kotlin/Native memory manager                          | Stable       | 1.9.20                   |                                                                            |
-| klib binaries                                         | Stable       | 1.9.20                   |                                                                            |
+| klib binaries                                         | Stable       | 1.9.20                   | Not including cinterop klibs, see below                                    |
 | Kotlin Multiplatform                                  | Stable       | 1.9.20                   |                                                                            |
 | Kotlin/Native interop with C and Objective C          | Beta         | 1.3                      |                                                                            |
+| cinterop klib binaries                                | Beta         | 1.3                      |                                                                            |
```

Comment:
I thought that it's called `C interop` in other docs. Am I wrong?

Reply:
I checked our docs, when talking about interop in general, we use 'C interoperability', but with specifics, we use 'cinterop tool', 'cinterop declarations', 'cinterop libraries'